package.json

```diff
@@ -28,12 +28,12 @@
     "build": "npm run preprocess && npm run bundle && npm run header && npm run stats",
     "cibuild": "npm run preprocess && node tasks/cibundle.js",
     "watch": "node tasks/watch.js",
-    "lint": "eslint --version && eslint . || true",
-    "lint-fix": "eslint . --fix",
+    "lint": "eslint --version && eslint .",
+    "lint-fix": "eslint . --fix || true",
     "docker": "node tasks/docker.js",
     "pretest": "node tasks/pretest.js",
     "test-jasmine": "karma start test/jasmine/karma.conf.js",
-    "citest-jasmine": "karma start test/jasmine/karma.ciconf.js",
+    "citest-jasmine": "CIRCLECI=0 karma start test/jasmine/karma.conf.js",
```

Comment:
It would be better to fix whatever checks the `CIRCLECI` var. It's confusing to see `CIRCLECI=0` on CircleCI. (If it needs to be this way, can you add a comment?)